boards/others/esp32c3_supermini/esp32c3_supermini.dts

```diff
@@ -0,0 +1,91 @@
+/*
+ * Copyright (c) 2024 Arrel Neumiller
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/dts-v1/;
+
+#include <espressif/esp32c3/esp32c3_mini_n4.dtsi>
```

Comment:
Should this be `#include <espressif/esp32c3/esp32c3_fx4.dtsi>` instead? Just asking, I haven't checked.

Reply:
Yes, it should be Fx4 (it has internal flash). 
Another holdover from the template I used (esp32_rust).